automation/tests/authentication.spec.ts

```diff
@@ -1,17 +1,150 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, } from '@playwright/test';
+import { AuthenticationPage } from '../pages/authentication.page.ts';
+import dotenv from 'dotenv';
+dotenv.config();
+import { loginCredentials } from '../test-data/authentication_page_data.js';
+import { EmailAuthCredential } from 'firebase/auth/web-extension';
```

Comment:
Check if it appears here by accident

Reply:
Yes, by accident. Will take it out from here